fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/RewritingRuleSet.java

```diff
@@ -37,8 +37,9 @@
 @SuppressWarnings("java:S1452")
 public class RewritingRuleSet extends CascadesRuleSet {
     private static final Set<ExplorationCascadesRule<? extends RelationalExpression>> EXPLORATION_RULES = ImmutableSet.of(
-    // new PredicatePushDownRule(),
-    // new RemoveRangeOneRule()
+    //            new PredicatePushDownRule(),
```

Comment:
Could we revert this change just to avoid causing trouble with merging #3401 (not that that conflict is a particularly difficult one to address)?

Reply:
done.